extensions/kill-process/src/index.tsx

```diff
@@ -14,43 +15,61 @@ export default function ProcessList() {
   const shouldShowPID = preferences.shouldShowPID ?? false;
   const shouldShowPath = preferences.shouldShowPath ?? false;
   const refreshDuration = +preferences.refreshDuration;
+  const [sortByMem, setSortByMem] = useState<boolean>(preferences.sortByMem ?? false);
+  const [aggregateApps, setAggregateApps] = useState<boolean>(preferences.aggregateApps ?? false);
 
   const fetchProcesses = () => {
-    exec(`ps -eo pid,pcpu,comm | sort -nrk 2,3`, (err, stdout) => {
+    exec(`ps -eo pid,ppid,pcpu,rss,comm`, async (err, stdout) => {
```

Comment:
there is no await in the following fn, don't think this async is needed.

Reply:
I forgot to delete it, will remove it soon!